tgui/packages/tgui/interfaces/ListInputModal.tsx

```diff
@@ -169,11 +185,15 @@ const ListDisplay = (props, context) => {
             id={index}
             key={index}
             onClick={() => onClick(index)}
+            onDblClick={(event) => {
+              event.preventDefault();
+              act('submit', { entry: filteredItems[selected] });
+            }}
             onKeyDown={(event) => {
               const keyCode = window.event ? event.which : event.keyCode;
-              if (keyCode === KEY_ENTER && isValid) {
+              if (keyCode >= 65 && keyCode <= 90) {
```

Comment:
Why'd you remove KEY_ENTER?

If it's some IE thing, can you please make these constants ourselves so that these magic numbers are readable?

Reply:
enter is handled in the window content itself, and yes I can make these more readable (probably)